components/brave_wallet_ui/components/extension/connect-with-site-panel/connect-with-site-header/connect-with-site-header.tsx

```diff
@@ -0,0 +1,125 @@
+// Copyright (c) 2023 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import * as React from 'react'
+import { create } from 'ethereum-blockies'
+
+// Types
+import { SerializableOriginInfo } from '../../../../constants/types'
+
+// Utils
+import { getLocale } from '../../../../../common/locale'
+
+// Components
+import { CreateSiteOrigin } from '../../../shared'
+
+// Styled Components
+import {
+  StyledWrapper,
+  FavIcon,
+  Title,
+  TitleWrapper,
+  SiteName,
+  SiteURL,
+  MessageBox,
+  InfoIcon,
+  BackButton,
+  BackIcon,
+  GradientLine,
+  LinkIconCircle,
+  LinkIcon
+} from './connect-with-site-header.style'
+import { AccountCircle } from '../select-account-item/select-account-item.style'
+import { HorizontalSpace, Column, Row } from '../../../shared/style'
+
+interface Props {
+  originInfo: SerializableOriginInfo
+  isReadyToConnect: boolean
+  onBack: () => void
+  address?: string
+  isScrolled: boolean
+}
+
+export const ConnectWithSiteHeader = (props: Props) => {
+  const { originInfo, address, isReadyToConnect, isScrolled, onBack } = props
+
+  // Memos
+  const orb = React.useMemo(() => {
+    return create({
+      seed: address?.toLowerCase() ?? '',
+      size: 8,
+      scale: 16
+    }).toDataURL()
+  }, [address])
+
+  return (
+    <>
+      <TitleWrapper
+        isScrolled={isScrolled}
```

Comment:
Maybe follow-up to change this to a scroll percentage? That way we can partially fade the element when partially scrolling

Reply:
Ditto on issue created: https://github.com/brave/brave-browser/issues/29299